packages/lit-auth-client/src/lib/providers/BaseProvider.ts

```diff
@@ -161,40 +165,43 @@ export abstract class BaseProvider {
 
         let response: SignSessionKeyResponse;
 
+        // common data for the signSessionKey function call
+        const commonData = {
```

Comment:
💖 Nice, is a lot easier to follow the differences between EthWallet and other branches now

Reply:
Seconded